spec/features/publish/courses/new_tda_course_spec.rb

```diff
@@ -494,5 +516,77 @@ def course_name_and_code
     course.decorate.name_and_code
   end
 
+  def given_i_am_on_the_check_answers_page_of_a_new_tda_course
+    given_i_am_authenticated_as_a_school_direct_provider_user
+    and_the_tda_feature_flag_is_active
+    and_i_visit_the_courses_page
+    and_i_click_on_add_course
+    and_i_choose_a_secondary_course
+    and_i_select_a_subject
+    and_i_choose_an_age_range
+    and_i_choose_a_degree_awarding_qualification
+    and_i_choose_the_school
+    and_i_choose_the_study_site
+    and_i_choose_the_applications_open_date
+    and_i_choose_the_first_start_date
+    then_i_am_on_the_check_your_answers_page
+  end
+
+  def when_i_visit_the_course_outcome_page
+    publish_course_confirmation_page.details.outcome.change_link.click
+  end
+
+  def and_i_choose_qts
+    publish_courses_outcome_edit_page.qts.choose
+    and_i_click_continue
+  end
+
+  def and_i_choose_fee
+    choose 'Fee - no salary'
+    and_i_click_continue
+  end
+
+  def and_i_choose_salaried
+    choose 'Salary'
+    and_i_click_continue
+  end
+
+  def and_i_choose_part_time
+    uncheck 'Full time'
+    check 'Part time'
+    and_i_click_continue
+  end
+
+  def and_i_choose_to_sponsor_a_student_visa
+    choose 'Yes'
+    and_i_click_continue
+  end
+
+  def and_i_choose_to_sponsor_a_skilled_worker_visa
+    choose('course_can_sponsor_skilled_worker_visa_true')
+    and_i_click_continue
+  end
+
+  def then_i_see_the_correct_attributes_in_the_database_for_fee_paying
+    course.reload
+    expect(course.study_mode == 'part_time').to be(true)
+    expect(course.funding_type == 'fee').to be(true)
+    expect(course.can_sponsor_skilled_worker_visa == false).to be(true)
+    expect(course.can_sponsor_student_visa == true).to be(true)
+  end
+
+  def then_i_see_the_correct_attributes_in_the_database_for_salaried
+    course.reload
+    expect(course.study_mode == 'part_time').to be(true)
+    expect(course.funding_type == 'salary').to be(true)
+    expect(course.can_sponsor_skilled_worker_visa == true).to be(true)
+    expect(course.can_sponsor_student_visa == false).to be(true)
```

Comment:
Same as the other ones

Reply:
https://github.com/DFE-Digital/publish-teacher-training/pull/4223/commits/ffdba5f9fd1845929fa2589593a4a60115137382